JamGame/Assets/Scripts/TileUnion/PlaceCondition.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Common;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using TileBuilder;
+using TileUnion.Tile;
+using UnityEngine;
+
+namespace TileUnion
```

Comment:
```suggestion
namespace TileUnionPlaceCondition
```
(because `RequiredTile` path looks like `TileUnion.RequiredTile` for now)